packages/repack/src/commands/bundle.ts

```diff
@@ -23,10 +24,16 @@ export async function bundle(
   config: Config,
   args: BundleArguments
 ) {
-  const webpackConfigPath = getWebpackConfigPath(
+  let webpackConfigPath = getWebpackConfigPath(
     config.root,
     args.webpackConfig
   );
+
+  if (webpackConfigPath.endsWith('.mjs') && Os.platform() === 'win32'){
+    webpackConfigPath = 'file:\\' + webpackConfigPath;
+    args.webpackConfig = 'file:\\' + args.webpackConfig;
```

Comment:
rewriting `args.webpackConfig` is quite unexpected. Why do we need to do that? You can pass an absolute path to the CLI argument yourself.

Also `getWebpackConfigPath` should be responsible for providing us correct `webpackConfigPath` for any platform, so we don't need to override it (and repeat the code in `start` command). Appending the `file:\\` doesn't sound like an elegant solution here. Can you think about better way to achieve the correct path for Windows?

Reply:
I know that this is not the best solution. I'll try to figure out, any hint?

Reply:
@RafikiTiki I'd appreciate your help here